core/src/main/java/org/ldaptive/handler/CaseChangeEntryHandler.java

```diff
@@ -245,4 +267,109 @@ public String toString()
       "attributeValueCaseChange=" + attributeValueCaseChange + ", " +
       "attributeNames=" + Arrays.toString(attributeNames) + "]";
   }
+
+
+  /**
+   * Creates a builder for this class.
+   *
+   * @return  new builder
+   */
+  public static Builder builder()
+  {
+    return new Builder();
+  }
+
+
+  /** Case change entry handler builder. */
+  public static final class Builder
+  {
+
+    /** Case change entry handler to build. */
+    private final CaseChangeEntryHandler object = new CaseChangeEntryHandler();
+
+
+    /**
+     * Default constructor.
+     */
+    private Builder() {}
+
+
+    /**
+     * Makes this instance immutable.
+     *
+     * @return  this builder
+     */
+    public Builder freeze()
```

Comment:
It strikes me that the object should be implicitly frozen upon calling `build()`, which would